src/main/java/org/mule/service/http/impl/service/client/GrizzlyHttpClient.java

```diff
@@ -462,6 +472,15 @@ private CompletableFuture<HttpResponse> sendAsync(HttpRequest request, Request g
     return future;
   }
 
+  /**
+   * To execute after handling redirect.
+   *
+   * @param redirectRequest the {@link Request} involved in the redirect.
```

Comment:
If some exception is found during the execution of the redirect, this argument will be null. (add this clarification)